### ISSUE TITLE
Quick select creation from table in treeview

### ISSUE THREAD

**Issue by adipose** [HeidiSQL/HeidiSQL#604]
## Current behavior

Cannot quickly create a query from a table, as you can in Workbench, DBVIS, and SSMS.

## Possible solution

Add a right click option or a "select" icon next to the tables in the treeview, to "select top 1000 rows" or something.

See here:

https://www.heidisql.com/forum.php?t=19296

**Comment by vinnyjames** [comment]
Here's a zoomed in example of workbench's quick select:

![image](https://user-images.githubusercontent.com/5167824/55745449-1a4e3e00-59ec-11e9-8457-b63e66d300b4.png)

**Comment by rentalhost** [comment]
The icon could be a good one.

**Comment by smalos** [comment]
Yes, I also miss having an option to quickly create a query from a table. Wouldn't it be straightforward to integrate this into the context menu when right-clicking on a table or view? A "Select Top 1000 Rows" menu item would be incredibly useful and in line with what's available in other tools !

**Comment by ansgarbecker** [comment]
Next build has a new context menu item "Select top 1000 rows", in the left database tree and in the database tab. The latter one supports multiple tables or views to be selected and queried:

![Image](https://github.com/user-attachments/assets/ea41ffd8-1b7c-41f6-ac54-9a32a554a168)

**Comment by adipose** [comment]
Amazing

**Comment by ansgarbecker** [comment]
Hope this now makes sens: the value of 1000 are now synced with the init setting for the data grid (which is also 1000 by default, I have set it to 5000 in the screen):

![Image](https://github.com/user-attachments/assets/7efbf601-95c3-4e00-98ac-9d40222d825e)

**Comment by adipose** [comment]
Yes makes sense. Is that formatted to locale settings?

**Comment by ansgarbecker** [comment]
Yes
